File: Javascript/CS133J/assortment/fizzbuzz/fizzbuzz.js
function action1() {

	var outstr = '';
	for (var i = 0; i <= 100; i++) {
		if( (i % 3 == 0 || i % 3 == 3) && (i % 5 == 0 || i % 5 == 5) ){
			outstr += i +' FizzBuzz' +'<br>';
		}
		else if(i % 3 == 0 || i % 3 == 3){
			outstr += i +' Fizz' +'<br>';
		}
		else if(i % 5 == 0 || i % 5 == 5){
			outstr += i +' Buzz' +'<br>';
		}
		outstr += i + '<br>';
	}
	document.getElementById('output').innerHTML = outstr;

}

File: python/fizzbuzz.py
def fizzbuzz(start, end):
	for x in range(start,end+1):
		if (x%3 or x%5) == 0:
			print('%d Fizzbuzz' % x)
		elif x%3 == 0:
			print('%d Fizz' % x)
		elif x%5 == 0 :
			print('%d Buzz' % x)
		else:
			print('%d' % x)
	return

fizzbuzz(1,100)